functions/dw_populate/fn_populateSheds.sqf

```diff
@@ -0,0 +1,125 @@
+﻿//exec: server
+
+params["_center", "_sheds"];
+
+_count = 0;
+
+if(!_sheds) exitWith {_count};
+
+// Demi abri avec mur en béton
```

Comment:
Shouldn't you make 3 different files for each type ?

Reply:
I should, and I will. However those scripts are being rewritten due to the occupation system feature dev. Once polished I will replace these scripts with the new ones.

Reply:
Resolution through #16